packages/store/gas-report.json

```diff
@@ -293,29 +293,41 @@
     "name": "get total of PackedCounter",
     "gasUsed": 27
   },
+  {
+    "file": "test/Schema.t.sol",
+    "test": "testEncodeDecodeSchema",
+    "name": "initialize schema array with 6 entries",
+    "gasUsed": 856
+  },
   {
     "file": "test/Schema.t.sol",
     "test": "testEncodeDecodeSchema",
     "name": "encode schema with 6 entries",
-    "gasUsed": 5639
+    "gasUsed": 2814
   },
   {
     "file": "test/Schema.t.sol",
     "test": "testEncodeDecodeSchema",
     "name": "get schema type at index",
-    "gasUsed": 185
+    "gasUsed": 127
   },
   {
     "file": "test/Schema.t.sol",
     "test": "testGetNumDynamicFields",
     "name": "get number of dynamic fields from schema",
-    "gasUsed": 74
+    "gasUsed": 68
+  },
+  {
+    "file": "test/Schema.t.sol",
+    "test": "testGetNumFields",
+    "name": "get number of all fields from schema",
+    "gasUsed": 34
   },
   {
     "file": "test/Schema.t.sol",
     "test": "testGetNumStaticFields",
     "name": "get number of static fields from schema",
-    "gasUsed": 85
+    "gasUsed": 79
```

Comment:
you may note that getting both dynamic and static is 2x cheaper than getting them separately. This is wrong and related to https://github.com/latticexyz/mud/issues/1267
I wonder if these tests should be removed to avoid confusion?